plugins/catalog/src/components/EntityLayout/EntityLayout.test.tsx

```diff
@@ -13,22 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React from 'react';
-import { EntityLayout } from './EntityLayout';
+import { CatalogApi } from '@backstage/catalog-client';
+import { Entity } from '@backstage/catalog-model';
 import {
   AlertApi,
   alertApiRef,
   ApiProvider,
   ApiRegistry,
 } from '@backstage/core';
+import { catalogApiRef, EntityContext } from '@backstage/plugin-catalog-react';
```

Comment:
would probably be convenient to re-export the `CatalogApi` type from the `-react` package.